src/lib/PriorityTools.ts

```diff
@@ -39,4 +39,29 @@ export class PriorityTools {
     public static priorityNameUsingNormal(priority: Priority) {
         return PriorityTools.priorityNameUsingNone(priority).replace('None', 'Normal');
     }
+
+    /**
+     * Get the {@link Priority} value from a string. A lower-case string is expected.
```

Comment:
How about lower-casing the input text, to remove the risk that calls don't spot this instruction, and just back None for everything?

Reply:
I thought about this but didn't implement it since my intention was to capture the current behaviour without change. 

I've updated the PR.

Reply:
Please let me know when/if this all good for you, I will rebase on top of the main.

Reply:
> Please let me know when/if this all good for you, I will rebase on top of the main.

Thanks Ilyas.
Now that I've got today's release done, yes please do rebase.
I'll then make a couple of minor wording edits and merge...

Reply:
Done